src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,112 @@
+import { Person } from '../types/Person';
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PersonLink } from '../components/PersonLink';
+
+const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [loadingError, setLoadingError] = useState(false);
```

Comment:
Naming convention can be different from project to project but in general, we name boolean variables with prefix `is` or `has`
```suggestion
  const [isLoading, setIsLoading] = useState(false);
  const [hasError, setHasError] = useState(false);
```